routes/survey.js

```diff
@@ -102,32 +104,34 @@ router.get('/:id/', async (req, res) => {
  * Queries postgres for current, previous, and compare data for a given
  * profile type, set of geoids and compare geoid. Joins the data, and adds
  * 'change' and 'difference' calculation values.
- * @param {string} profileName - The profile type
+ * @param {('acs'|'decennial')} survey - The type of survey to return results for. Must be 'acs' or 'decennial
  * @param {Array} geoids - The list of geoids for the given selected geography
  * @param {string} compareTo - Integer string representing the geoid of the comparison geography
  * @returns {Object}
  */
-async function getProfileData(profileName, geoids, compareTo, db) {
+// async function getProfileData(profileName, geoids, compareTo, db) {
+async function getSurveyData(survey, geoids, compareTo, db) {
   const isAggregate = geoids.length > 1;
 
-  const queryBuilder = getQueryBuilder(profileName);
-
+  const queryBuilder = getQueryBuilder(survey);
+  console.log(queryBuilder(geoids))
+  
   // get data from postgres
-  const [rawProfileData, rawCompareData, rawPreviousProfileData, rawPreviousCompareData] = await Promise.all([
-    db.query(queryBuilder(profileName, geoids)),
-    db.query(queryBuilder(profileName, [compareTo])),
-    db.query(queryBuilder(profileName, geoids, /* is previous */ true)),
-    db.query(queryBuilder(profileName, [compareTo], /* is previous */ true)),
+  const [rawProfileData, rawCompareProfileData, rawPreviousProfileData, rawPreviousCompareProfileData] = await Promise.all([
```

Comment:
why add `Profile` token into these variable names? `Profile` here, like in `rawProfileData`, refers to the data for the selected geography. `rawCompareData` refers to data for the comparison geometry.
Actually I think it might be better to rename these as `rawSelectionData`, `rawCompareData`, `rawPreviousSelectionData`, `rawPreviousCompareData`

Reply:
I think you accidentally re-included `Profile` here